src/App.js

```diff
@@ -0,0 +1,42 @@
+import { React, useEffect } from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+} from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import NavBar from './Navbar';
+import HomePage from './components/HomePage/home';
+import ConfirmedCases from './components/ConfirmedCases/ConfirmedCases';
+import DeathsCases from './components/Deaths/Deaths';
+import RecoveredCases from './components/Recovered/Recovered';
+import OpenCases from './components/OpenCases/OpenCases';
+import './App.css';
+import lastDayNumbers from './redux/LATEST/thunk/thunk';
+import dayBeforeNumbers from './redux/DAYBEFORE/thunk/thunk';
+import getHistoricalData from './redux/HISTORICalMONTHDATA/thunk/thunk';
+
+function App() {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```